src/components/pages/dashboard.js

```diff
@@ -1,13 +1,33 @@
-import React from "react";
+import React, { memo } from "react";
+import Button from "@material-ui/core/Button";
+import { useHistory } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
 
-const Dashboard = props => {
+const useStyles = makeStyles({
+    dashboardWrapper: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        paddingTop: '40px',
+    },
+});
+
+
+const Dashboard = () => {
+  let history = useHistory();
+  const classes = useStyles();
   return (
-    <div>
-      <div>
-        <h1>Dashboard</h1>
-      </div>
+    <div className={classes.dashboardWrapper}>
+      <h1>Dashboard</h1>
```

Comment:
use Typography for displaying text in material-ui
`<Typography variant='h1'>Dashboard</Typography>`